server/index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+const express = require('express');
+
+const app = express();
+const port = 3003;
+
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const cors = require('cors');
+const path = require('path');
+// const router = require('router.js');
+// const db = require('../db)
+
+// middleware
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
you won't be needing nested objects?

Reply:
ooh that was a typo on my point, thanks!